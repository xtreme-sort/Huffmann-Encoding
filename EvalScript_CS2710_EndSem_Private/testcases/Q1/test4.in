BuildFrequencyTable
wKzCHXxw_Mkq99elDXChcZRvjICkjEOb0tZQSUtgzlfAOnUqn9qwZuBkxJ0b710hq0zAjrI5ImxqkG0VHx0U4L2gvFxiXUv__WjSZlqJNBJ5_lpNa1s3fucPA1EYHYbb0lKvPZRRHBlLLD5L0zw_nAgTYfkYjopKPmnMAZ5ylRLbzkEJ6OqdZBmiNLIfM0lQmJ_YxdOy71fSDrcsUrH28J5IbQVhJX8uTfhhsMCMpP_rfWCBfiUXDuxiimfSUL3YdsTU_SIPb4j83NINvcY0iYfOrYl3Uk35k8BAcVfvz3giumyRXi1nJ9aPqmUquNHFnbmXMBd2efErd_5VbdGtGBmakxKULTd8q_9Kg5eJML_4MlNl68bLXJ8hhFe4JGJKHkoNkNfo1KdKFKXFGR3nKoJsRT6Y4Rbqyn5jZ_qtZLDWb0S_H4V3bRuudgEXOJgEF3Cl5eroNLHo6laQknAQu5dLHLJydkByW3a6EPdj2Sz0xEnZvOusuLl4gltsrdvCfXp4ZgXXKxt9V1BP2_zaCkeD2o2mePcKig7mhgwopOj5d6em1JTK_a2vgMPxJ5L9sAfNbMBjNaQ30MU43ddO0SrVgjU7IqPszvPDc1k4GA4CBOxBuTZMaKJ2_U1sYBgfP9S77m3blEhOaXlbwcgOxOdrOqGqDQCq50uaGRut68rWeXEuhZKw6sTvkcviBhIUd4iluGXSqp0akOGkxqFCXmv43k7ej176NzODMz82KmsxtCgeGzULHCmrFERtKuOYAPUnCsuXDpieQrvLobOLXWvWebNfR_hAUM0CJ96hBkFCJjjZm0coBw1M_fNTw99F6H7MbJ5t7IH44mVBmno8C5HDt24dDa7sh9yVWXZNg_bgC6iTDmpuQehF7kDgsFlUI7fjhYHNiKkFrhO1dKiECmMin2_AbmbpwbDUhcQvoIOG8p9sBXb_4CYw7114bp16s9fmLLdGlMOh8mxD96lppLYJH4fxwYsdl3ZE5X0XcugMorN28iRLZqLIXVI8AW5HdcxJl6o8F9oa6utDgmpF5V7L5FpMScHcconGidlnp_QNGKVZYxuP79I9lJmBMnqDPc0NEOrPiz_1tTSDYV2uRHfIT6hGD5p9XDFtO9e6nec6ZPqzNmZDmvgAJRCMZE916ZqHqEkRIaUGoc0_tjLslpLec3OCknw98Gy0NXO_Dv0oWByE9R6ByxJeVkcHXHuDeX_ZhsgBhxzVQePOcV3z0Bx9ruT7wlqouCWz9Gw_TSvfntIW16AfJEy_5nSLhSfbDtHuJfQlj01__21Ei1TFNDVrFZ3q85FipgqBbWvRqSJPqOCYJeBPO1E3EC7uP2BCKNLkMA0JFcDLdoyiQOr8cvXARaFM7EndF36Tm9Jscebp3DdFXJdQYzwpjwrFDjvrD5aqU669lkBEAY3UcgkcMoAa2HCJ8JUJG6Mp6YirFE1A0KLKgZzBF61A2RMDHNMsJTStFNE6rpkAYQDaRu4xfRgpfCh7iw0bgOcRjSa2NEKqtoznRnIPxXuNYDKJzhrBxGZxnL4kj6tTO6maCGYzzRsfxmCcAIynGIf9RTQGEuqYeFD7Zj_p9MGjw4LRyPktjYn7sdgzZN9LbvWwdayuwOrDg9441qGJi6avX9GWFnQSzOYEgChO1luMZAYd8bH1TLpz3tHHyLrCER_SUSIB_tgtiDSCthe796zsBhwHPo8mDsZwsVPbleB8nra9aJ7KoEC6HF1QxnG9RZLBrXa793eCrrcYd2s_qwsFnB1TAB4f5AJFPGWOPl4p2ynFCB2VwOzCA0YavqDiiEMvDYv7G4AqJ9PdjGY4RowR3yd3VlJEvenbpttNdNrIQpob1pcVty0XOYW_0nGawJdQ0kUkGtLOGgFuY50aOw1B61ZfMGQ2x2LMc0Y_8jWQSxp9zG2hBp_5KSWucovYS332t_SfI3Z2F8QjOjuR7K2Dx6n1mPqDJzFqIHhJEta0wwWu3QiPih4a_bKmNDMrjg4m00cVBtch_jHQzKYL1ijjlM3TAjQzdGFd_cCVqX0CR_RS9uIz4iZo1iqrLbJIIbmHZvkwcHuwWYw0iYrs5x2cfV5WeL8_jgW_qk5ZRmBCFhRYl6PMl9juFsSPq33DW3rXVPvF8JbTIjwFM0d4n0rA9T9At4WWDb4Aq3TjFlOv6JFJ4cLkT3UTTiZU01oHB_yuOkh7VjGJPRFo_LSGC_Pg4Y5FXEFCrcoPd6W_Ftp6haGPlU2uPvH6eGnLOQ_K1Av4zRb0HwNL_Gi2eVZi9qnpixYtfslXOAU3HLOAoFxvW5nuwgkR5eWNuNHbnTMrCBlERMgu4GXfjSjN5jVHv0PCS_ZyEuJbm_9FOoKNp6xDpimaLEIgptH0eDK_fnicrryO_rIgwcIR94UWk0uw1fUt4AXSlrp_H6iH5NpJV3SGwFfHCoLEp07OCTa_EQcYyEl6afDEOsl1_lWAkizeU4UplY3mfRmKnSCZ8IRcBYpOG6TeDpOnhk5zi8htP5F9065jmlFb7cbCNQ4eOTDnhu31lFyHCEnaQanzmkAVpDmorkzzT3FPvwPodxmmCnP0WlpsHjfP6oJU_ud8aVA8QUgC3pHaS_I9a7eCXhj9GkGBUBhPeHBph48U5rZYZpaStVstFGcvVgSloU53WxOwaYwqI063nlu49NHYzJJN3nSS_QpdB7hHTgZ2h4qJSoIgmdJpnbenIPu2FhTSXPxFxal4Fc2v5bSWBwpPawmVGwDGxpVs6a00SopFT_RC97Vc0BG9l2TqDaWL5gkDrHYzbqjSwxBGg5bgdSvncXGqxGlHLCZ0JXi3Um5QmuQ_bLgbcEY6FEHQJgf6GueNIP6JK4grwOxehQASMjj2naA_pZholdM7AYnX9iG1KCuLC82h6nu1RlBRjx1uTIzMiEkHwevjq4PmnfmVY2KTo9cny3fXAwduGybFKrFi19HfAIxQwQR8ApcWD8jHYYqhEtF9pgoAtLoRqcJYowuAaKEsBkDuhvocXKpGNnlkNN_NATdCMDHk14GNamytNXsV4PYpIpa0Tx25d73ETYmjqth1N4gw44kV6QA2rWh86ugQcn03kf7Z0PhCTJDAOvfkFOl87DKWTKOSZ_AY0yZBlXkypNNBWbZC7xwjkcN88RhmLCUT4FcYWtZsK6GWIMSHsNyWDG9EFAVE5DQLbuqU1vYsP8XCI3yOFCYapGq8IdZ4to4euhSLZ_Gmm86iCxFFe2exbtX44H8TGQy9enuLx0FhubX5DXsW5Q6XCUWGSAtPDGAk3XCSvvO6P5mK1ozkmukTpW1mjhtonSuyoWJjENWO3CqrsQeymbGiE31UDzxfkc2U2xjjmezantTbjST0smIzKMaMM7ZyMa6aiXZMSHK7xMq9B4BIOqFLL2b49hxjRdgOyUePfvfOYp4jxnkRpM2b6CONF2Ou3Pf9BkxuIy_FB8n2m14M7k6PMBIbGsKWoBAepXvPhRdM8LI3zR2p74YI24RWGt6GKJS2X7jO7Ph7CItHCMLrH834e5A4VHgfnV_N2S4mpIcQ9tKvDbT7rCIGRkca5EDnsUjBb_yIlnkUXYzMr6IAucBxSRzaAaRmKA4gQg9iZfIZtDlPttmwjZjakd4Qy_285Nf2wtzQchleNFXxv_qeX005guHsqQaYNd87TThDm2BxTM5rZ8dVH5QazEZWaOnkxK9ZvGkRynW9FyF0lUyhDfnk1kikX1VaMX7JL7PoLfYtvII7DoVgDNdQgwd_aJs5LqmfB0q_vg8ecCae6uuAk9mFOmEOD8YXUqb9HCQmnbFsM0t2iNd8kfnWUobv6ayPQcGljRq0pCO7lE3rjZakyduyrBxG2fnb2YhuL0a_lpFWZHXrSXsd_i41AF8DHhSA51ga0jZmkWkBKX_0Po1yY1R6fEo5mbB0ZkPU9uEx3F8ClxpStxDmX7OYPPu_pNQ1tl_TlBtaWoTIl7bOd1ZEd_Jay5JNdAYUmlfBs43DTpHqrkTnwVRdrBnOihdv0rNg3xiAlJtwGljriDTkVHg3ryZ59lvj9X197Xgmrawa5VNYLROdQhCtDYDAJ0AabtzxYOaY4sPdCgsGWALb38OS6oVd2B0WjVfM54OpQKMUESw756slHm0Li1sO_SLzSct9DNvijj3etmxDGG5hm6LwHCMePEjOS0yc5vQFQm4lwxWHC3fwYvQ6ftFmeu_Jj5pXXmxWMzMr9wvcccd1LZFEjy7MjNskXakPNLbf0GVDVw4U1zD1mpvUdW7IbksLhSyC9UaoqpVBhXWV5nCkgomfg5y7297Ocl3IjUr01nis4GtQDuP4LZ2RKNoWqrpkB8OaTrjzQiw4V_t0mseqaQg_8B79Yn_MSVITHkUnZC6rlz_Zz1efENWWqbSzlAd3htoYf03e3tHHY44bBcvSIDFGu6j5ZGUAAxMZxxY14d537KBdNKw9Hy1pZ06o8xm_mubxdZOJYYnIRvipnhEeCnZvRRdBNF8gF3_6cKiSPyKWFsc8ErONyytJbw02a6dnQQ5Aznv0OXoWZqfRzS7vfOIER8NgwVw5al2TC5NaiS1HevtS5VMLE5j5bV_lSD8V2rTVcF7zu1tOvdRHbAesi4w9UbTIiz0eHjOE9FGkAk6TqOHD0VzuZcKFYjZQY7rxeGYhQHsKToeLcB5YJoKaAuwhvUcgTOUNq6m40YCJPrmhH6q9rWKzr7nnTiqh_T0s0dqnFYZafGmheCY7VLHjafeSH4KodU7UR2ofxNDkmLAtPOdhIoV47lsbsBJPYwrSPmOAcVpzaj0ZPGckKaSqxR0h4NnSbXfeoeu6TJXrAcqdnnxDIaAtCz9BPnwSYXTDo42LmmSWgVoJZRz6lLgkszXBx9So0p3H9Uzovq6Y3t7kLP3icsIUq0JihU1hT9l65h7WJiW9ou5gVCphzRmfjUQqqjhXB5LYTK1no4ep4l61lCGP5JuWkHdRS3xhXLJIqoOja3vdaMjQwHZxHnbMRIkHsmw4wSNMyRicEqopKs_tGEMv9jIEGPWSgsdQ664PDrQU5WdRu2KqkBaNLxNoObpCvIyoAtwPagm3Faz5hK8w5tBlysk7jxTPCiT1kSC9qpenwytFtdZvXXMQZ6JpS1n7YGDWA7yMMttisEXYQq5PiN1Popr37xW9fWB2S1TDSHIAtyZkjhA9ixG6tvakjUKChKeB2gBhvgyEIkduaH_K8TNvSll6Slsp66f6dScI36TWlMn5ffT5CMOjVrPnaaNEvJ1oecPKJf_eAS11M5xv6XeyCVHhht3GRCuhFddWnzpFtrfuzyISJYBheYo1095tZV2EfV3aWQl3NWi19YmGAqvB_WNGk96ft7QPbiaPT8mLjNG0VNb_ux71YQ0o782fKKgPvq7etpfWSd3RikTQ2HXIIbUvYWxUfaaG03GUzpFIurqNBnxay45qCYMRzWWpMx2rHQaO1WHsrxmD1nrZ0VHtlZTxtfJBZ3YnV1kuAfXZxVyp49VtM3_EmzM9B6t8A3TWv25aZAq_b3mzlJ7lhmKCMAaan5d94R6nMz4GLhCNCYPDTRsgUcTr8pQkmnRWKrJe06VGJJDnHNU4vzWbqEJpAAFGiqs_jW88dtU7ouNk51Qhamf_mIJGafN9ePu_HK9kCFCEQeYWAsTf7ovTNfZiS5D30Y2fG4otTJ4k3o2iNIEF0JmUruTiKTKF7PhW8ggLTRAwI_50626XWc0I4XEL60tM1P0b1lPpLTNcn_1m_UelEGYlMUjGQPy8LjJlEq5AwpS53EKUVTj6r83341z7dlews_HQcXbkDcJtEDc7HYTnhamyil2ukiehZEwEOcs6DAV3zn88lPpKcrwRQR9Hwsa7hqXzAXrq0JFl6xDf1GEmy5HbgDrT9lsHIjcAwdWREpKdEQThXGtqRtm55hxnKFGwVGroxToYQCyoYotgAGw7VCet5CsgpjXnqeTNgC3mxOXIISmwUnseVXPVtGV2iU7dFCr0u2Bq4q0EwR90fcTpR7hK6V3CXv8341pg6eOUSmMDE7FHYzsBXsyjhVCahDb3FKSuyaWdHtbqapjSeHnb1x8uXLJ0VMWX2qRqbonuE2FW98xOjmrPUw5RpeBlzsuPcYlKAPZtJaojxghIfgBO8zSpqKkhJQq0wPC302ycEoajqtd8AYNvTOTU5Hvjv0U4OUCjtPP9aEOi_froZRt3D0fCs_uN7myDhiDLoHbOQwzMgoeDeldb4CUJpYnfY3uyW9YteO1IdbjsMBV2eQ4g8BYrUAmK4EfLHFqy68QoIVoEba2FxuqQSBUbHh6yQRkKcHke9cOc1_peC1eZXQQCiitYw4HNn9xs4oZq1CGRM6pyl0aY9kErpkEVSGyDbIvrKV4HcvgVDmXgJR5xJZRlXapxtJzN6hdgUMjIkngk1KPNht6flrCgKaaG0FLwllwmP8YAOIm1nAux1I1fNHCfQXlRoKmK4NmBq0xOfOZEt5X7F18WcHB_JZB0JCjrgGk9ZinnXkgQm3tB1xb0eUJzghmM88vsEPwC1tinWSfcWOZQlFIql9OjYbQ2HhelUsGlaqzZDCaOtf4ogW3BmbMLQdH979TcewcfazeAq2c0HV0Dsltw4H7mGMX6AHjF3_nCLV7TLjfUOoOnqiaY2MOfeKPwfWsiYEIBJdkiUUAJIz6ItQgs2Q3RiSSyImA0afHArLo6Y5VJU_yFXCjAsUvvAUnlSNt_5dMl_lxD8Pm1Li0tfGBjJM2jEgHfxuXgvhrfS666PQaMzQfgU09b5bojJgdiVufpa7mYOoiJ9798bDzgfx0NwT5jEIR7Xk8Wk9QPVWX7HlE2b1Qx41N4RGEoiqJ8CGuY43gnF8yzVoDNUHG8kRmm9RkpL0y5BW3iuGTd5bvl8kEMuZC0o9kxB1furH36P2GiRdpIYMT8WbqXBMYazAfMzps_wniRxPrrLxx5eBMaZDQlh0_YgXynJjWPZOVe_EO0X1vq1nd9kf9vyudbuT1dZCDCd3Clnp68JRfBrRNJ0SgGb7RIVkldbXZslfxvMxyl_XisdlfLksdSaqfmkMTbqiJWiCwqGIG07EBfbSi7tOU3ASwsDzJysYRQJVUYihRSQW19TsubkzcnJ7IuEM0KNRiaZcGRM_kXdlDrJmxEmHDGo1q7bLvCiM5QON11Cfj0DfU5qvBby8XbK7yGPeZkkX0dSOktWSGQv9JFgjAP_aL7tbzqlmUg8VFrvsHlsw83eMqE47rLSFxcEsWAgemDrM7FNeHBw8luKFJDjLIiHDjbMs8rKQwsPypf2gLHAeWzyQufKQF7UK_KOIHx7dyMWZzeQgd0146ZqV4QD8B3SeoP2FCrm3zDD7qmB31Fge8Fp7ebupm3om3iDMZye43JtVVPLrseL33hRaMyDqcZjwtUNCpnY2Iw3Iy_V2zc1ZLjnWkgbPzg1aZ1pS8OYpKobl5nipN_Lnhw6StK59Jn3eeCi2pOBVtAhOvaBG6BzOChEPcKbDR0sXCZ8mEe0IdybrKFZYDOM2tiHDr6xCEh44hnJv3QBSz4ijlJQQH7br7Y0lKtMpd5wT9ftWu7
Quit
