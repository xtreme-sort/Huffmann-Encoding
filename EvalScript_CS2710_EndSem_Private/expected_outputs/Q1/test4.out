0 136
1 122
2 97
3 121
4 122
5 117
6 123
7 114
8 103
9 121
A 115
B 121
C 136
D 130
E 115
F 126
G 130
H 134
I 109
J 130
K 107
L 112
M 117
N 114
O 127
P 123
Q 118
R 114
S 120
T 109
U 108
V 107
W 116
X 120
Y 135
Z 118
_ 120
a 136
b 127
c 111
d 121
e 118
f 131
g 118
h 121
i 115
j 125
k 132
l 135
m 142
n 134
o 114
p 125
q 126
r 115
s 109
t 132
u 120
v 105
w 112
x 121
y 97
z 107
